jwst/resample/resample_step.py

```diff
@@ -148,15 +147,13 @@ def get_drizpars(self, ref_filename, input_models):
             self.log.error("No row found in %s matching input data.", ref_filename)
             raise ValueError
 
-        # Define the keys to pull from drizpars reffile table.  Note the
-        # step param 'weight_type' is 'wht_type' in the FITS binary table.
+        # Define the keys to pull from drizpars reffile table.
         # All values should be None unless the user set them on the command
         # line or in the call to the step
         drizpars = dict(
             pixfrac=self.pixfrac,
             kernel=self.kernel,
             fillval=self.fillval,
-            wht_type=self.weight_type,
```

Comment:
Is wht_type still present in the ref file, but we're just ignoring it? So what happens if a user intentionally tries to change the behavior by modifying the values in the ref file?

Reply:
We are ignoring it.  All the default reffiles in CRDS have this set as `exptime`, and it would necessarily override whatever I set as the default in `ResampeStep.spec`.  So we are ignoring it.  And we are going to be deprecating replacing `drizpars` reffiles with `pars-resamplestep` reffiles anyway, so I really don't want to create a new drizpars reffile for every instrument or encourage users to do so either.  It's a dead end.